packages/table/src/TableHeadCell.ts

```diff
@@ -58,6 +58,45 @@ export class TableHeadCell extends SpectrumElement {
     @property({ attribute: 'sort-key' })
     public sortKey = '';
 
+    protected handleKeydown(event: KeyboardEvent): void {
+        const { code } = event;
+        switch (code) {
+            case 'Space':
+                event.preventDefault();
+                this.addEventListener('keyup', this.handleKeyup);
+                break;
+            /* c8 ignore next 2 */
+            default:
+                break;
+        }
+    }
```

Comment:
Keydown, in previous consumptions, in this way was meant to allow for `this.active = true` while the `Space` key was doing to better mimic the interaction you get with a `<button>` element. This element currently includes styles for this interaction, but not the property to power those styles. We probably should...

Can you add this to the beginning of the class:
```
@property({ type: Boolean, reflect: true })
public active = false;
```

`this.active = true;` here. And `this.active = false;` to the `handleKeyup()` logic? Then we should be good to go here.

Reply:
@jnurthen Can you please make sure this is added as requested above!

Reply:
I think I did in https://github.com/adobe/spectrum-web-components/pull/4473/commits/550b46a5320f5baf36887dd2e9e6957a5ef6f062